packages/react-components/react-toast/src/components/Toaster.tsx

```diff
@@ -19,27 +20,42 @@ export type ToasterProps = Partial<ToasterOptions>;
 
 export const Toaster: React.FC<ToasterProps> = props => {
   const { offset } = props;
-  const { getToastsToRender, isToastVisible, toasterRef } = useToaster<HTMLDivElement>(props);
+  const { getToastsToRender, isToastVisible } = useToaster<HTMLDivElement>(props);
+  const announceRef = React.useRef<Announce>(() => null);
+
+  const announce = React.useCallback<Announce>((message, options) => {
+    announceRef.current(message, options);
+  }, []);
```

Comment:
Signatures of `annouce` and `announceRef` are the same, is a reason to have `annouce` separately?

Reply:
to avoid passing a ref to children